pytest_docker_postgres/__init__.py

```diff
@@ -65,3 +78,14 @@ def db_engine(in_docker_compose, docker_services):
     create_database(url)
     yield create_engine(url)
     drop_database(url)
+
+
+@pytest.fixture(scope="function")
+def schema_db_engine(db_engine, load_database):
+    with db_engine.connect() as conn:
+        sql_files = sorted(glob.glob(os.path.join(load_database, "**/*.sql"), recursive=True))
+        for file_path in sql_files:
+            with open(file_path) as file:
+                with conn.begin():
+                    conn.execute(text(file.read()))
```

Comment:
```suggestion
                    conn.connection.cursor().execute(file.read())
```

and I think then you don't need the `transaction` anymore

Reply:
transactions are inevitable. :stuck_out_tongue_closed_eyes: